Paintroid/src/main/java/org/catrobat/paintroid/tools/helper/DefaultNumberRangeFilter.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Paintroid: An image manipulation application for Android.
+ * Copyright (C) 2010-2021 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.paintroid.tools.helper
+
+import android.text.Spanned
+import android.util.Log
+import org.catrobat.paintroid.ui.tools.NumberRangeFilter
+
+class DefaultNumberRangeFilter(private val min: Int, override var max: Int) : NumberRangeFilter {
+    companion object {
+        private val TAG = DefaultNumberRangeFilter::class.java.simpleName
+    }
+
+    override fun filter(
+        source: CharSequence,
+        start: Int,
+        end: Int,
+        dest: Spanned,
+        dstart: Int,
+        dend: Int
+    ): CharSequence? {
+        try {
+            val input = (dest.toString() + source.toString()).toInt()
```

Comment:
Hmm, in my opinion, this logic is not default 😅  (also not the original one)
@ThorstenBandel could you please have a look at this?

Reply:
It is a little bit of a hack, @CodeChamp-SS if you find a better way to program this feel free to do so, but as it works as intended its not too urgent to be changed